### PR TITLE
inets: replace calls `size/1` to `XXX_size/1`

### PR DESCRIPTION
**Pull request by kikofernandez**
Replace calls in the `inets` module from `size/1` to `tuple_size/1` or `byte_size/1`.
This PR is part of issue (PR) #6669

### CODE REVIEW

File: lib/inets/src/http_client/httpc.erl [hunk 1]
```diff
@@ -1337,7 +1337,7 @@ validate_ipfamily(BadValue) ->
     bad_option(ipfamily, BadValue).
 
 validate_ip(Value) 
-  when is_tuple(Value) andalso ((size(Value) =:= 4) orelse (size(Value) =:= 8)) ->
+  when is_tuple(Value) andalso ((tuple_size(Value) =:= 4) orelse (tuple_size(Value) =:= 8)) ->
```
Comment (bjorng) [inline lib/inets/src/http_client/httpc.erl:1340]:
```suggestion
  when tuple_size(Value) =:= 4; tuple_size(Value) =:= 8 ->
```

File: lib/inets/src/http_lib/http_transport.erl [hunk 2]
```diff
@@ -403,11 +403,11 @@ peername({essl, _}, Socket) ->
     do_peername(ssl:peername(Socket)).
 
 do_peername({ok, {Addr, Port}}) 
-  when is_tuple(Addr) andalso (size(Addr) =:= 4) ->
+  when is_tuple(Addr) andalso (tuple_size(Addr) =:= 4) ->
```
Comment (bjorng) [inline lib/inets/src/http_lib/http_transport.erl:406]:
```suggestion
  when tuple_size(Addr) =:= 4 ->
```

File: lib/inets/src/http_lib/http_transport.erl [hunk 3]
```diff
@@ -403,11 +403,11 @@ peername({essl, _}, Socket) ->
...
     PeerName = ipv4_name(Addr), 
     {Port, PeerName};
 do_peername({ok, {Addr, Port}}) 
-  when is_tuple(Addr) andalso (size(Addr) =:= 8) ->
+  when is_tuple(Addr) andalso (tuple_size(Addr) =:= 8) ->
```
Comment (bjorng) [inline lib/inets/src/http_lib/http_transport.erl:410]:
```suggestion
  when tuple_size(Addr) =:= 8 ->
```

File: lib/inets/src/http_lib/http_transport.erl [hunk 4]
```diff
@@ -436,11 +436,11 @@ sockname({essl, _}, Socket) ->
     do_sockname(ssl:sockname(Socket)).
 
 do_sockname({ok, {Addr, Port}}) 
-  when is_tuple(Addr) andalso (size(Addr) =:= 4) ->
+  when is_tuple(Addr) andalso (tuple_size(Addr) =:= 4) ->
```
Comment (bjorng) [inline lib/inets/src/http_lib/http_transport.erl:439]:
```suggestion
  when tuple_size(Addr) =:= 4 ->
```

File: lib/inets/src/http_lib/http_transport.erl [hunk 5]
```diff
@@ -436,11 +436,11 @@ sockname({essl, _}, Socket) ->
...
     SockName = ipv4_name(Addr), 
     {Port, SockName};
 do_sockname({ok, {Addr, Port}}) 
-  when is_tuple(Addr) andalso (size(Addr) =:= 8) ->
+  when is_tuple(Addr) andalso (tuple_size(Addr) =:= 8) ->
```
Comment (bjorng) [inline lib/inets/src/http_lib/http_transport.erl:443]:
```suggestion
  when tuple_size(Addr) =:= 8 ->
```

File: lib/inets/test/httpd_bench_SUITE.erl [hunk 6]
```diff
@@ -643,7 +643,7 @@ do_handle_request(CB, S, {gen, Data}, Opts, KeepAlive) ->
 send_file(CB, S, {gen, Data})  ->
     CB:send(S, Data);
     %% ChunkSize = 64*1024,
-    %% case size(Data) of
+    %% case byte_size(Data) of
```
Comment (bjorng) [inline lib/inets/test/httpd_bench_SUITE.erl:646]: I don't see any `is_binary/1` test preceding this call.

**Review by bjorng — COMMENTED** [review]
You don't need the `is_tuple/1` test when you use `tuple_size/1`. Otherwise looks good.

**Review by bjorng — APPROVED** [review]

**Review by IngelaAndin — APPROVED** [review]

**Pull request merged**